source/api_cc/src/DeepPot.cc

```diff
@@ -712,6 +712,7 @@ compute (ENERGYTYPE &			dener,
     assert (nloc_real == atommap.get_type().size());
 
         nlist_data.copy_from_nlist(lmp_list);
+        nlist_data.shuffle_exclude_empty(fwd_map);
```

Comment:
#2111 should be merged before this PR so that the change to this line will disappear in this PR.